src/core_modules/capture-core/utils/reactQueryHelpers/query/useMetadataQuery.types.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import type { UseQueryResult } from 'react-query';
+
+export type CustomOptions = $ReadOnly<{|
+    useCache?: boolean,
+    enabled?: boolean,
+|}>;
+
+export type Result<TResultData> = UseQueryResult<TResultData>;
```

Comment:
Instead of returning just `data`, `loading` and `error`, I changed it to return all `useQuery`-attributes. Open to discussion on this one.